src/python/pants/backend/python/goals/tailor.py

```diff
@@ -90,6 +91,30 @@ def is_entry_point(content: bytes) -> bool:
     return _entry_point_re.search(content) is not None
 
 
+@rule_helper
```

Comment:
the `@rule_helper` annotation will soon be deprecated, might as well not use it already.

Reply:
Ah I see, thank you. I've kept it consistent with another rule helper doing a similar thing.

```
@rule_helper
async def _find_source_targets(
```

Perhaps we could refactor both when it's deprecated.